odata-protocol/11.4 Data Modification.md

```diff
@@ -1058,9 +1067,10 @@ of `deleted`, the value of `failedOperation` MUST be `delete`, otherwise
 be represented in the response as deleted entities annotated with the term
 `Core.DataModificationException` with a `failedOperation` value of
 `insert`.
-- Failed updates within the request SHOULD
+- Failed updates within the request MUST
```

Comment:
Why did we have "SHOULD" for updates?

I don't think we can leave that at SHOULD and allow services to silently ignore updates, especially if they "only" fail due to an outdated ETag.

Reply:
I suspect we were thinking that the inclusion of the entity with its current values was sufficient -- whether the values had changed because of server logic, an exogenous change, or an error, the client processing should be the same (to apply the current values). If there is an etag then it should be updated, so the client can retry the request.  So, it's not the end of the world if the service doesn't mark the returned entity with a data modification exception.

We can debate whether that logic holds, but I suspect that was our reasoning when we authored the original wording.